hera_cal/frf.py

```diff
@@ -497,6 +498,85 @@ def filter_data(self, data, frps, flags=None, nsamples=None,
             filt_flags[k] = f
             filt_nsamples[k] = eff_nsamples
 
+    def run_tophat_frfilter(self, to_filter=None, weight_dict=None, mode='clean',
+                            frate_standoff=0.0, frac_frate_sky_max=1.0, min_frate=0.025,
+                            max_frate_coeffs=None,
+                            skip_wgt=0.1, tol=1e-9, verbose=False, cache_dir=None, read_cache=False,
+                            write_cache=False,
+                            data=None, flags=None, **filter_kwargs):
+        '''
+        Interpolate / filter data in time using the physical fringe-rates of the sky. (or constant frate)
+        Arguments:
+          to_filter: list of visibilities to filter in the (i,j,pol) format.
+              If None (the default), all visibilities are filtered.
+          weight_dict: dictionary or DataContainer with all the same keys as self.data.
```

Comment:
can we use `wgts` for this? That is also more consistent with other usages of this kind of term

Reply:
Yep.